apps/api-extractor/build-tests.cmd

```diff
@@ -1,3 +1,3 @@
 @ECHO OFF
 @SETLOCAL
-rush build -t api-extractor-lib1-test -t api-extractor-lib2-test -t api-extractor-lib3-test -t api-extractor-scenarios -t api-extractor-test-01 -t api-extractor-test-02 -t api-extractor-test-03 -t api-extractor-test-04 -t api-documenter-test
+rush build -t api-extractor-lib1-test -t api-extractor-lib2-test -t api-extractor-lib3-test -t api-extractor-scenarios -t api-extractor-test-01 -t api-extractor-test-02 -t api-extractor-test-03 -t api-extractor-test-04 -t api-extractor-test-05 -t api-documenter-test
```

Comment:
Consider naming this test something more informative like `api-extractor-extra-spaces-dts` (happy to bikeshed) instead of `api-extractor-test-05`. That being said, I know there's precedent for the naming you chose given the current test names, so I don't feel strongly (maybe @octogonz does).

Reply:
I left it to follow the nomenclature in place, but I can definitely rename it to be more semantic.

Reply:
Gotcha, I'll leave it up to you.